src/config-mapper.js

```diff
@@ -1,11 +1,11 @@
 const fs = require('fs')
 const path = require('path')
 const toml = require('toml-j0.4')
-const packagePath = path.resolve(process.cwd(), './package.json')
-const servicePackage = fs.existsSync(packagePath) ? require(packagePath) : {}
 const Definition = require('./definition')
 
 function load (configFile) {
+  const packagePath = path.resolve(process.cwd(), './package.json')
```

Comment:
path.resolve does not need `./` in its parameters, and omitting that make it that much more reliable across operating systems.